.ci/setup.sh

```diff
@@ -22,8 +22,14 @@ elif [ "$ID" == ubuntu ];then
 	sudo apt-get -qq update
 	sudo apt-get install -y -qq make automake qemu-utils python-pip coreutils moreutils bc
 	sudo pip install yamllint
-else 
+else
 	echo "Linux distribution not supported"
 fi
 
+# yq needed to correctly parse runtime/versions.yaml
+if [ -z "$(command -v yq ${GOPATH}/bin/yq)" ]; then
+	echo "INFO: yq not found, installing..."
+	bash -c "$(curl -sSL https://raw.githubusercontent.com/kata-containers/runtime/master/.ci/install-yq.sh)"
```

Comment:
Yes. Since that script might need to move or get renamed at some point, we *could* do this instead:

- Add a `make install-yq` target in the *tests* repo.
- Get this script to call it.

The advantage of doing that being we hide the script behind a make target. And that would work as all repos (including osbuilder) already clone `tests.`

Reply:
For CI , I think what @jodh-intel proposed is fine.
For the rootfs creation instead, I'd prefer to curl the raw  `versions.yaml` and depend on the runtime repo [*].

---
[*]: This is because since recently, osbuilder can be installed with `make install-scripts`, to get a set of script that can be used to build and package a kata image. And I am using this to [build a kata image on OBS](https://github.com/marcov/containment-rpm-kata/blob/master/build_kata_img.sh). And I think others could benefit of this too :)